apps/src/studio/studio.js

```diff
@@ -3315,8 +3333,8 @@ Studio.setBackground = function (opts) {
 
     // Draw the tiles (again) now that we know which background we're using.
     if (level.wallMapCollisions) {
-      $(".tile_clip").remove();
-      $(".tile").remove();
+      $(".specificTileClip").remove();
```

Comment:
I don't understand the reason for separate tile classes. It seems like the `randomTile` ones will never be removed. Previously, a map could initialize with random tiles, and they would be replaced if the student called `setMap` (and would be updated if the `setBackground` changed the theme). Is there a reason that you changed this?